### PR TITLE
LPFG-890 Change status

### PR DESCRIPTION
**Pull request by mick-jones**
- Added publish/archive button to course overview page.
- Added content based on whether course is draft or published
- After a discussion with Abi regarding 2 different styles for the right hand form it was decided that the old version:
image 1
would be replaced by the newer version in the prototype:
image 2
This was to keep status separate from actions and to keep all versions of the form consistent

### CODE REVIEW

File: views/page/course/archive.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">Confirmation of archiving</h1>
+            <p class="govuk-body">Are you sure you want to archive this course?</p>
+            <p class="govuk-body">Archiving means this course can not be booked or completed. Where the course was completed prior to archiving, it will still appear in learning records.</p>
+            <form method="post" action="/content-management/courses/{{course.id}}/status">
+                <p class="govuk-body"><strong>Archiving a course a irreversible<strong></p>
+                <input type="hidden" name="status" value="Archived" />
```
Comment (m24murray) [inline views/page/course/archive.html]: `Archiving a course a irreversible` > `Archiving a course is irreversible`

File: views/page/course/course-details.html [hunk 2]
```diff
@@ -44,7 +44,8 @@ <h2 class="govuk-heading-m govuk-!-font-size-36">{{i18n.course.text.details}}</h
             </form>
             <a class="cancel-button govuk-body govuk-link" href="/content-management/courses/title">Cancel</a>
         </div>
-        <div class="govuk-grid-column-one-third"/>
+        <div class="govuk-grid-column-one-third">
+        <div>
```
Comment (Peter-D-Taylor) [inline views/page/course/course-details.html]: Shouldn't this be </div>?

**Comment by m24murray** [comment]
I wouldnt expect that you can edit an archived course on the content-management home screen. The `Edit` link takes you straight to the preview page with the edit banner:

image

image

I would like to run through this with Abi before signing off review.

**Comment by m24murray** [comment]
I wouldn't expect that user's can create/edit audiences for an archived course:

image

Again, please run past Abi.

**Comment by m24murray** [comment]
Sorry for the design/content hat here, but in my opinion, this description of what happens when a user archives a course is missing a lot of things.

From an admin perspective, the user should be made aware that they can no longer:
- edit a course (title, descriptions e.t.c)
- add any new modules (and possibly audiences)
- add any new events to current face-to-face/classroom courses

From a learner perspective, archived courses will:
- no longer appear in search results or suggestions for you
- be removed from required and other learning plans

image

File: views/page/course/course-details.html (hunk 2 above)
Comment (mick-jones) [reply views/page/course/course-details.html]: Good spot, fixed

File: views/page/course/archive.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+{% set banner = true %}
+{% set backButton = "/content-management/courses/" + course.id + "/overview" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">Confirmation of archiving</h1>
+            <p class="govuk-body">By archiving, you will no longer be able to amend this course in anyway. Nor can it be searched, booked or completed.</p>
```
Comment (m24murray) [inline views/page/course/archive.html]: `anyway` -> `any way`

File: views/component/courseEvents/template.njk [hunk 4]
```diff
@@ -10,7 +10,9 @@
     <div class="module-container-small">
         <div class="title-container">
             <p class="govuk-body no-margin"><strong>{{ module.title | default(course.title, true) }}</strong></p>
-             <a class="govuk-body govuk-link" href="/content-management/courses/{{ course.id }}/modules/{{ module.id }}/events/">Add new event</a>               
+            {% if course.staus !== "Archived" %}
```
Comment (m24murray) [inline views/component/courseEvents/template.njk]: what's `staus`?

File: views/page/course/archive.html (hunk 3 above)
Comment (mick-jones) [reply views/page/course/archive.html]: fixed

**Pull request merged**